src/services/clients/paymentApi.client.ts

```diff
@@ -0,0 +1,21 @@
+import 'reflect-metadata'
+
+import { CreatePaymentRequest, Payment } from 'ch-sdk-node/dist/services/payment/types'
+import { ApiResponse, ApiResult } from 'ch-sdk-node/dist/services/resource'
+import { inject } from 'inversify'
+import { provide } from 'inversify-binding-decorators'
+import APIClientFactory from './apiClient.factory'
+
+@provide(PaymentApiClient)
+export class PaymentApiClient {
+
+  public constructor(@inject(APIClientFactory) private factory: APIClientFactory) {}
+
+  public async createPayment(token: string, body: CreatePaymentRequest): Promise<ApiResponse<Payment>> {
+    const response: ApiResult<ApiResponse<Payment>> = await this.factory
+      .getPaymentService(token)
+      .createPaymentWithFullUrl(body)
```

Comment:
if you use createPayment instead of createPaymentWithFullUrl, can you remove https://github.com/companieshouse/dissolution-web/pull/55/files#diff-32e9dcceac6d3f5c5c321e8d8ab593e8R14 ? The SDK should handle the endpoints, we should just have to just provide the base url.

Reply:
I thought that at first. But when doing the `post`, I was getting a 404 back.
Then I noticed this in the ch-sdk-node, https://github.com/companieshouse/ch-sdk-node/blob/master/src/services/payment/service.ts#L17. `createPayment` is posting to `/payment` and not `/payments`.
I'm not sure if that's an error in ch-sdk-node, but that's why I had to use `createPaymentWithFullUrl`.

Reply:
I would reach out to one of the developers who added this functionality to confirm. Seems like a bug but there could be a good reason behind it.